pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1225,6 +1163,83 @@ func (l *LibvirtDomainManager) findOrCreateVirDomain(
 	return virDomain, err
 }
 
+func (l *LibvirtDomainManager) handleDisksAttachment(
```

Comment:
Can we find a better name for this?
Is it processing the disks hotplug? Does it contain all storage logic?

Reply:
`syncDiskHotplug` to follow with `syncNetworkHotplug` ?

This is SyncVMI() so the VM is to be created and I'm not sure hotplug fits well here.

Reply:
The question is if here only hotplug is processed.

In the network case, we process the network in the end for hotplug purposes. So I would just call it `networkHotplug` or with the `sync` prefix.

Reply:
> The question is if here only hotplug is processed.

You are asking because of the naming, correct? Hotplug is definetly processed here, looking at https://github.com/kubevirt/kubevirt/pull/4458 https://github.com/kubevirt/kubevirt/commit/9d7a0394eb26d9bf9d44d13f4f4441d3517d8414 https://github.com/kubevirt/kubevirt/commit/4cbbe5b9e52006ebf38afe183fda1544d6d10831 

For the non hotplug, isn't it handled in `preStartHook()` ?  I'm not 100% sure about this area. For the moment, I'll rename the function similar to the networkHotplug

Reply:
Yes, it makes sense. At the beggining all the "regular" configuration and domain start is processed, then the hotplug stuff continues.

Thanks.